tests/suites/test_suite_pk.function

```diff
@@ -434,19 +552,12 @@ exit:
  */
 mbedtls_svc_key_id_t pk_psa_genkey_ecc(void)
 {
-    mbedtls_svc_key_id_t key = MBEDTLS_SVC_KEY_ID_INIT;
-    psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
-    const psa_key_type_t type =
-        PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY_SECP_R1);
-    const size_t bits = 256;
+    mbedtls_svc_key_id_t key;
```

Comment:
This removed the initialization `= MBEDTLS_SVC_KEY_ID_INIT`. I don't think that was intentional, probably an accident in resolving conflicts. Please add the initializer back. (Multiple occurrences in this file.)